src/js/components/covid19/awardSpendingAgency/AwardSpendingAgency.jsx

```diff
@@ -56,13 +65,13 @@ const AwardSpendingAgency = () => {
     useEffect(() => {
         const params = {
             filter: {
-                def_codes: defCodes.map((defc) => defc.code)
-                // TODO - add award type codes
+                def_codes: defCodes.map((defc) => defc.code),
+                award_type_codes: awardSpendingAgencyTableTabs.filter((type) => type === activeTab).codes
```

Comment:
Award type codes could also come from the existing mapping [here](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/dataMapping/search/awardType.js#L70) (and can be excluded from the request to default to all award type codes)